### DISCUSSION TITLE
关于邮箱通知关闭

### DISCUSSION THREAD

**Discussion by hsuehyaung** [share121/inter-knot#1753] [其他]
<h2 style="color: red;">  
    <a href="https://blog.csdn.net/ylb0109/article/details/104549130" style="color: red; text-decoration: none;">找的方法</a>  
</h2>

**Comment by hhhhhz222** [comment]
#1361：绳匠们回复完帖子后记得点下Unsubscribe按钮，否则一段时间后你的邮箱就99+啦

**Comment by SWYH34827** [comment]
好的，正需要，谢谢

**Comment by likeseele** [comment]
感谢大佬

**Comment by Yith123** [comment]
大佬666
